test/utils/helpFn.ts

```diff
@@ -71,7 +66,7 @@ export const getContext = async (polkadotApi: ApiPromise) => {
  
            const event = _.find(events, ({ event }) => polkadotApi.events.system.ExtrinsicSuccess.is(event));
            if (event) {
-             resolve({ extrinsicHash: txHash, blockHash: status?.asFinalized?.toString() });
+             resolve({ extrinsicHash: txHash, blockHash: status?.asFinalized?.toString(), events });
```

Comment:
Do you want to pass in all `events` or a selected `event` at this line? The `if(event)` condition doesn’t look necessary to me.

Reply:
In this context, "events" is a **return** value that includes all the events generated by the extrinsic.

I think the `if(event)` condition is necessary. It checks whether the extrinsic execution was successful.

Reply:
Okay, got it. Could you add these comments to the line above the `if(event)` to explain it?